src/RequestHeaderParser.php

```diff
@@ -29,7 +29,7 @@ public function feed($data)
             try {
                 $this->parseAndEmitRequest();
             } catch (Exception $exception) {
-                $this->emit('error', [$exception]);
+                $this->emit('error', [$exception, $this]);
```

Comment:
What would you suggest? Remove it here and from line 21, which would create a minor BC break. Or something else?